main.py

```
@@ -6,7 +6,7 @@
 import sentry_sdk
 import torch
 
-from fastapi import FastAPI, Request, Query, Depends, Security
+from fastapi import FastAPI, Request, Query, Depends, Security, HTTPException
```

Ruff (0.11.9)

`fastapi.Security` imported but unused

Remove unused import: `fastapi.Security`

(F401)